src/database/queries/tag.js

```diff
@@ -0,0 +1,21 @@
+import knex from '../knex'
+import * as _ from './utilities'
+
+const add = attributes =>
+  _.createRecord( 'tag', attributes )
+
+const deleteAll = () =>
+  _.deleteAll( 'tag' )
+
+const getAll = () =>
+  _.findAll( 'tag' )
+  .orderBy('names', 'asc')
+
+const getBy = ( column, data ) =>
+  _.findAllWhere( 'tag', column, data )
+
+const expunge = ( column, data ) =>
+  _.deleteRecord( 'tag', column, data )
+
+
+export { add, deleteAll, getAll, getBy, expunge }
```

Comment:
I think it would be easier to use these functions if they were given tag specific names `addTag`, `deleteAllTags`, `getAllTags`, `getTagBy`, `deleteTag`

Also you do not need this export at the end if you just put `export` in front of each `const`